calicoctl/calico_ctl/pool.py

```diff
@@ -122,19 +122,58 @@ def ip_pool_add(cidrs, version, ipip, masquerade):
         print "IP in IP not supported for IPv6 pools"
         sys.exit(1)
 
-    # TODO Reject any cidrs that overlap with existing cidrs in the pool
-    pools = []
-    for cidr in cidrs:
+    current_pools = client.get_ip_pools(version)
```

Comment:
Lots of variables with similar names and purposes floating around here. Since this guy is only used to create `current_pool_nets`, why not just overwrite current_pools?
```
current_pools = client.get_ip_pools(version)
current_pools = [IPNetwork(pool.cidr) for pool in current_pools]
```

Better yet, could one line it:
```
current_pools = [IPNetwork(pool.cidr) for pool in client.get_ip_pools(version)]
```